webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/logger/ConfigurationStatusLogger.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ *************************************************************************************
+ * Copyright 2017 Normation SAS
+ *************************************************************************************
+ *
+ * This file is part of Rudder.
+ *
+ * Rudder is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * In accordance with the terms of section 7 (7. Additional Terms.) of
+ * the GNU General Public License version 3, the copyright holders add
+ * the following Additional permissions:
+ * Notwithstanding to the terms of section 5 (5. Conveying Modified Source
+ * Versions) and 6 (6. Conveying Non-Source Forms.) of the GNU General
+ * Public License version 3, when you create a Related Module, this
+ * Related Module is not considered as a part of the work and may be
+ * distributed under the license agreement of your choice.
+ * A "Related Module" means a set of sources files including their
+ * documentation that, without modification of the Source Code, enables
+ * supplementary functions or services in addition to those offered by
+ * the Software.
+ *
+ * Rudder is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
+
+ *
+ *************************************************************************************
+ */
+package com.normation.rudder.domain.logger
+
+import com.normation.NamedZioLogger
+
+/**
+ * Applicative pure logger for configuration status that may have vary in Rudder, 
+ * from success to errors in different configuration management parts of Rudder
+ */
+object ConfigurationStatusLoggerPure extends NamedZioLogger {
```

Comment:
it should be a sublogger of the generation one I believe, more than of status

Reply:
the thing is, the generation loggers start with `policy.`

I thought that I would write logs under the same logger for node properties errors, but I was not convinced that all node properties error and technique compilation errors should be logged with the `policy.` namespace...

(Also there is already [a `node.properties` logger](https://github.com/Normation/rudder/blob/branches/rudder/8.2/webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/logger/ApplicationLogger.scala#L144), maybe this should be the one used for node properties errors ?)

Reply:
Also I was already think about a separate web page that would have all the global information about policies, errors in nodes, techniques, etc. and I would have named it "Configuration status", that is why I chose the same name for the logger

Reply:
Loggers are by applicative units. A "status" logger makes sense if we have a part of rudder dealing with statuses. 

So ok, let say we have status as an application unit, different from policy generation, and that it deals only with the status related logic, not what the status is for. 

Still, you should have either a general `status`, or a `status.techniques` (with a `status.generation` and a `status.properties`), and not configuration, that is the whole thing of managing configurations.